etc/make_release.py

```diff
@@ -342,87 +355,65 @@ def build_c_driver(c_driver_install_dir, c_driver_build_ref, quiet):
     otherwise return None.
     """
 
-    if not quiet:
-        click.echo('Building C Driver (this could take several minutes)')
     mongoc_prefix = os.path.abspath(c_driver_install_dir)
+
+    if not quiet:
+        click.echo(f'Building C Driver at {mongoc_prefix} (this could take several minutes)')
```

Comment:
Since you're using a format string, you probably need to supply a value to replace `{mongoc_prefix}`.

Reply:
This is a Python3 "f" string. The `{mongoc_prefix}` is a direct reference to the `mongoc_prefix` variable.

Reply:
OK.  Got it.  For some reason I misread the documentation and somehow came away with the notion that the values had to specified in the same line.